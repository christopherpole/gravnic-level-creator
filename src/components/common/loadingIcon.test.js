import React from 'react';
import { configure, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import LoadingIcon from './loadingIcon';

configure({ adapter: new Adapter() });

describe('Loading icon', () => {
  it('Renders without exploding', () => {
    const tile = shallow(<LoadingIcon />);

    expect(tile).toHaveLength(1);
  });

  it('Matches the current snapshot', () => {
    const tile = shallow(<LoadingIcon />);

    expect(toJson(tile)).toMatchSnapshot();
  });
});
