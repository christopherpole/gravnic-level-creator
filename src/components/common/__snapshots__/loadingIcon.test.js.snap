// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading icon Matches the current snapshot 1`] = `
<styled.div>
  Loading goes here
</styled.div>
`;
